feat(website): Partner announcement (#13314)
cloudquery/cloudquery: website/pages/blog/announcing-aws-partnership.mdx

---
title: Announcing CloudQuery as an Official AWS Partner
tag: announcement
date: 2023/08/24
description: >-

>   CloudQuery has been recognized as an Amazon Web Services Partner and validated via a foundational technical review.  This Partner program will enable CloudQuery to build and service more customer needs including security, cost optimization, infrastructure, and development.

Reviewer: ```suggestion
  CloudQuery has been recognized as an Amazon Web Services Partner and validated via a foundational technical review. This Partner program will enable CloudQuery to build and service more customer needs including security, cost optimization, infrastructure, and development.
```

author: jsonkao
---

import { BlogHeader } from "../../components/BlogHeader"

<BlogHeader/>

> We are excited to announce that CloudQuery has officially been recognized as a [member of the AWS Partner Network](https://partners.amazonaws.com/partners/0010h00001kNW07AAG).  From this partnership program, we're looking forward to continued development on extracting, transforming, and loading data from AWS APIs to a variety of supported destinations that include databases, data lakes, and streaming platforms for further analysis.  CloudQuery supports multiple use cases built on top of this data platform including security, cost optimization, infrastructure, and development.

Reviewer: ```suggestion
We are excited to announce that CloudQuery has officially been recognized as a [member of the AWS Partner Network](https://partners.amazonaws.com/partners/0010h00001kNW07AAG). From this partnership program, we're looking forward to continued development on extracting, transforming, and loading data from AWS APIs to a variety of supported destinations that include databases, data lakes, and streaming platforms for further analysis. CloudQuery supports multiple use cases built on top of this data platform including security, cost optimization, infrastructure, and development.
```

![AWS Partner Badge](/images/blog/announcing-aws-partnership/partnerbadge.png)

> We have a exiting roadmap ahead of new features and improvements.  Stay tuned!

Reviewer: ```suggestion
We have a exiting roadmap ahead of new features and improvements. Stay tuned!
```

> We have a exiting roadmap ahead of new features and improvements.  Stay tuned!

Reviewer 2: ```suggestion
We have an exciting roadmap ahead of new features and improvements.  Stay tuned!
```

Author: Thanks!  Fixed.

## Highlights

* CloudQuery was recently featured on an AWS live streamed data show, Let's Talk About Data: https://www.twitch.tv/videos/1894215221

* Part of the partner process involved us working with AWS to complete multiple steps including an AWS Foundational Technical Review to ensure our offerings meet AWS's stringent technical requirements and provide value to customers. 

## How CloudQuery works with Amazon Web Services

> CloudQuery supports a variety of [source plugins](https://www.cloudquery.io/docs/plugins/sources/overview) and [destination plugins](https://www.cloudquery.io/docs/plugins/destinations/overview).  By using these source plugins and destinations, information from AWS configurations and deployments can be synced.  Additionally, infrastructure data from a variety of sources can be synced to AWS-related and hosted data destinations.

Reviewer: ```suggestion
CloudQuery supports a variety of [source plugins](https://www.cloudquery.io/docs/plugins/sources/overview) and [destination plugins](https://www.cloudquery.io/docs/plugins/destinations/overview). By using these source plugins and destinations, information from AWS configurations and deployments can be synced. Additionally, infrastructure data from a variety of sources can be synced to AWS-related and hosted data destinations.
```

### Destinations

CloudQuery supports the following AWS-related destination plugins:

* [Amazon Kinesis Firehose](https://www.cloudquery.io/docs/plugins/destinations/firehose/overview)
* [Gremlin (AWS Neptune)](https://www.cloudquery.io/docs/plugins/destinations/gremlin/overview)
* RDS Compatible databases including [PostgreSQL](https://www.cloudquery.io/docs/plugins/destinations/postgresql/overview), [MySQL](https://www.cloudquery.io/docs/plugins/destinations/mysql/overview).
* [S3](https://www.cloudquery.io/docs/plugins/destinations/s3/overview)

### Policies and Dashboards

CloudQuery supports additional compliance frameworks including:
* AWS Cost Optimization
* AWS CIS Benchmarks
* AWS PIC DSS
* AWS Foundational Security Best Practices
* Custom security frameworks including publicly available resources in AWS
* Unused Resources in AWS
* Kubernetes 

Please see our links for policies and dashboards for more information about these offerings:
* [Policies](https://www.cloudquery.io/docs/core-concepts/policies)
* [Dashboards](https://www.cloudquery.io/docs/core-concepts/dashboards)

## Use Cases

> CloudQuery supports multiple use cases including security, cost optimization, infrastructure, and development.  Below is a sampling of the use cases that can be built with CloudQuery with AWS infrastructure data:

Reviewer: ```suggestion
CloudQuery supports multiple use cases including security, cost optimization, infrastructure, and development. Below is a sampling of the use cases that can be built with CloudQuery with AWS infrastructure data:
```

* [Open Source Cloud Security Posture Management (CSPM)](https://www.cloudquery.io/how-to-guides/open-source-cspm)
* [Open Source Attack Surface Management with Neo4j (ASM)](https://www.cloudquery.io/how-to-guides/attack-surface-management-with-graph)
* [Analyzing AWS Cost and Usage Data](https://www.cloudquery.io/how-to-guides/query-aws-cost-and-usage-athena)
* [Managing AWS Resource and Usage Limits](https://www.cloudquery.io/how-to-guides/aws-resource-limits-and-service-quotas)
* [Managing Resilience Strategy with AWS Backup](https://www.cloudquery.io/how-to-guides/manage-resilience-with-aws-backup)

## Get Started

> We'd love to hear your thoughts.  We can be reached on [GitHub](https://github.com/cloudquery/cloudquery/issues), [Discord](https://www.cloudquery.io/discord).

Reviewer: ```suggestion
We'd love to hear your thoughts. We can be reached on [GitHub](https://github.com/cloudquery/cloudquery/issues), [Discord](https://www.cloudquery.io/discord).
```
